### DISCUSSION TITLE
Quarto in combination with graet_tables containing math: formatting does not work

### DISCUSSION THREAD

**Discussion by ed3o3tst** [quarto-dev/quarto-cli#10959] [Q&A]
### Description

Hi all,

I'm trying to make tables for an manuscript in Quarto. I will export the manuscript in PDF and HTML so I hope I can make it work in both formats. If it will work too in Word that's an extra bonus, but guess my co-workers can work around that. ;)

So what I am trying to do is to make a table with the Python package [great_tables](https://posit-dev.github.io/great-tables/articles/intro.html). This table include math symbols. When I format my table in for example Jupyter Notebook then all goes well. However if I render the jupyter notebook with quarto it does not get the math symbols and so it just displays $var_1$ and $$var_2$$ as rownames. This is a very minimal example

```
---
title: "Short example"
format: 
  html: default
  docx: default
  pdf: default
---

```{python}
import pandas as pd
from great_tables import GT, md, html, style, loc

df = pd.DataFrame([['$var_1$',1],['$$var_2$$',2]],columns=['var','p'])
GT(df)
```

Is there a way to work around this? Thanks in advance!

Im using:
- Quarto version 1.5.57
- great_table version 0.12.0

---------------------------------------------------------
Edit: I just figured out that for HTML output I can make it work if I include 

```{=html}
<script>
  MathJax = {
    tex: {
      inlineMath: [['$', '$'], ['\\(', '\\)']]
    },
    svg: {
      fontCache: 'global'
    }
  };
</script>
<script type="text/javascript" id="MathJax-script" async
  src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js">
</script>
```

That's fine. But can't get it working for PDF. Maybe there is a similar trick that I am not aware of?

**Comment by cderv** [comment]
cc @rich-iannone 

Are you supporting this for Quarto ? (through `data-qmd` attribute maybe ?)
